pkg/internal/codegen/parse/index.go

```diff
@@ -183,61 +115,47 @@ type resourceTags struct {
 	ShortName string
 }
 
-// ParseResourceTag parses the tags in a "+resource=" comment into a resourceTags struct
-func parseResourceTag(tag string) resourceTags {
-	result := resourceTags{}
+//parseResourceAnnotationValue is a helper function to extract resource annotation.
```

Comment:
nit: space missing between `//` and `parse...`

Reply:
Done!